server/express.js

```diff
@@ -157,14 +159,23 @@ function makeApp(authAddress, cdapConfig, uiSettings) {
       helmet({
         contentSecurityPolicy: {
           directives: {
-            imgSrc: [`'self' data: ${cspWhiteListUrls}`],
+            imgSrc: [`'self' data: ${cspWhiteListUrls}`, 'www.googletagmanager.com', 'https://www.gstatic.com', 'https://ssl.gstatic.com', 'https://www.google-analytics.com'],
+            connectSrc: [`'self'`, 'https://www.google-analytics.com', 'https://ampcid.google.com'],
+            fontSrc: [`'self'`, `'unsafe-inline'`, 'https://fonts.gstatic.com', 'data:'],
             scriptSrc: [
               (req, res) => `'nonce-${res.locals.nonce}'`,
               `'unsafe-inline'`,
```

Comment:
nit: Can we open a JIRA to track an improvement to remove `unsafe-inline` and `unsafe-eval` down the road?

Reply:
@njbriggs https://cdap.atlassian.net/browse/CDAP-20666